playground/nextjs/package.json

```diff
@@ -11,8 +11,13 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@clerk/nextjs": "beta",
-    "@clerk/themes": "beta",
+    "@clerk/backend": "file:.yalc/@clerk/backend",
+    "@clerk/clerk-react": "file:.yalc/@clerk/clerk-react",
+    "@clerk/clerk-sdk-node": "file:.yalc/@clerk/clerk-sdk-node",
+    "@clerk/nextjs": "file:.yalc/@clerk/nextjs",
+    "@clerk/shared": "file:.yalc/@clerk/shared",
+    "@clerk/themes": "file:.yalc/@clerk/themes",
+    "@clerk/types": "file:.yalc/@clerk/types",
```

Comment:
🔧 i think we should revert this since it's being used in the preview deployment.
cc: @LekoArts

Reply:
@LekoArts Do we need this? This gets overwritten everytime we execute the yalc command.